packages/@best/api-db/src/store.ts

```diff
@@ -69,6 +69,7 @@ export const saveBenchmarkSummaryInDB = async (benchmarkResults: BenchmarkResult
         benchmarkResults.map(async (benchmarkResult) => {
```

Comment:
reset file permissions back to `100644`.  Same for the other files that have `100644 → 100755` in the filename of the review